civictechprojects/static/css/LogInController.css

```diff
@@ -33,3 +33,8 @@
   background-color: #dddddd;
   cursor: no-drop;
 }
+
+.LogInController-forgotPassword {
+  text-align: center;
+  font-size: 11px;
+}
```

Comment:
`text-align: center;` doesn't seem to be needed here.
 
I'd suggest raising the font size. 11px is smaller than anything else on the page, it's small enough I have readability concerns, and it's going to be a tough mobile clicktarget. 

One option would be to move it inline with but aligned right in the `Password:` div above the password box, which would let us keep it at 16px (default) size but also keep it out of the way for people who don't need it.
Something like this would work:
```
<div style="clear: both;">
  <span style="float: left;">Password:</span>
  <span style="float: right;"><a href="#">Forgot password?</a></span>
</div>
```
(But without inline styles.)